platform/linux/config.cmake

```diff
@@ -42,10 +39,8 @@ macro(mbgl_platform_core)
         target_sources(mbgl-core
             PRIVATE platform/linux/src/headless_backend_egl.cpp
         )
-        target_link_libraries(mbgl-core
-            PUBLIC -lGLESv2
-            PUBLIC -lEGL
-        )
+        mason_use(swiftshader VERSION 2017-11-20)
+        target_add_mason_package(mbgl-core PUBLIC swiftshader)
```

Comment:
Wouldn't it be better to have this under a flag? Removing EGL/mesa linkage takes hardware acceleration away from the Linux build.

Reply:
Discussed in chat that linking against SwiftShader still links libEGL.so/libGLESv2.so, so the resulting output can be used to run with a hardware-accelerated version of Mesa as per the original comment.